Makefile.third

```diff
@@ -317,6 +317,25 @@ ifdef ANDROID
 	readelf -d $@ | grep "no dynamic" \
 		|| echo "warning: Android L+ support only PIE binary and can't run this"
 endif
+# ===========================================================================
+# dropbear
+
+$(OUTPUT_DIR)/dropbear: $(THIRDPARTY_DIR)/dropbear/CMakeLists.txt
+	install -d $(DROPBEAR_BUILD_DIR)
+	cd $(DROPBEAR_BUILD_DIR) && \
+		$(CMAKE) -DCC="$(CC)" \
+		-DLIBS="$(if $(or $(ANDROID),$(WIN32),$(DARWIN)),,-lrt) $(if $(ANDROID),-static,)" \
```

Comment:
We also want something like

```
-DCPPFLAGS="$(CFLAGS) -I$(ZLIB_DIR)"
```

and

```
-DLDFLAGS="$(LDFLAGS) -L$(ZLIB_DIR)"
```

Or in some kind of LIBS form. I don't know what dropbear wants precisely.